index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
+      <input id="name" type="text" value="Badru Rohit" placeholder="input name"><label id="name-label"></label>Name</input><br>
+      <input id="email" type="email" value="
+rohitmurari@gmail.com"placeholder="input email"><label id="email-label"></label>Email</input><br>
+<input id="number"  min="5" max="50" placeholder="input" ><label id="number-label"></label> age</input><br>
+<select id="dropdown"> <option value="volvo">1</option>
+  <option value="2">2</option>
```

Comment:
Please try adding spaces between the different logical code blocks so that it is readable and more organised.
Please kindly indent the code two spaces respectively to make the code look neat